package.json

```diff
@@ -1,70 +1,70 @@
-    "prettier": "^2.8.8",
-    "prettier-plugin-svelte": "^2.10.0",
-    "sass": "^1.62.1",
-    "svelte": "^3.55.0",
-    "svelte-check": "^3.3.2",
-    "svelte-preprocess": "^5.0.3",
-    "svelte2tsx": "^0.6.14",
-    "tslib": "^2.5.2",
-    "typescript": "^5.0.4",
-    "vite": "^4.3.8"
-  },
-  "peerDependencies": {
-    "ag-grid-community": "^28 || ^29 || ^30",
-    "svelte": "^3"
-  },
-  "scripts": {
-    "dev": "vite dev",
-    "build": "vite build",
-    "package": "svelte-package",
-    "preview": "vite preview",
-    "test": "playwright test",
-    "check": "svelte-check --tsconfig ./tsconfig.json",
-    "check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
-    "lint": "prettier --check --plugin-search-dir=. . && eslint .",
-    "format": "prettier --write --plugin-search-dir=. .",
-    "release": "commit-and-tag-version"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/MichaelKim/ag-grid-svelte.git"
-  },
-  "keywords": [],
-  "author": "Michael Kim",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/MichaelKim/ag-grid-svelte/issues"
-  },
-  "homepage": "https://github.com/MichaelKim/ag-grid-svelte#readme",
-  "files": [
-    "dist"
-  ],
-  "types": "./dist/index.d.ts",
-  "main": "./dist/index.js",
-  "svelte": "./dist/index.js",
-  "exports": {
-    ".": {
-      "types": "./dist/index.d.ts",
-      "svelte": "./dist/index.js"
-    }
-  }
+	"name": "ag-grid-svelte",
+	"version": "0.3.1",
+	"description": "An unofficial Svelte wrapper for AG Grid",
+	"type": "module",
+	"devDependencies": {
+		"@playwright/test": "^1.34.0",
+		"@sveltejs/adapter-static": "^2.0.2",
+		"@sveltejs/kit": "^1.20.4",
+		"@sveltejs/package": "^2.0.2",
+		"@typescript-eslint/eslint-plugin": "^5.59.6",
+		"@typescript-eslint/parser": "^5.59.6",
+		"ag-grid-community": "^30.0.0",
+		"ag-grid-enterprise": "^30.0.0",
+		"commit-and-tag-version": "^11.2.1",
+		"eslint": "^8.41.0",
+		"eslint-config-prettier": "^8.8.0",
+		"eslint-plugin-svelte3": "^4.0.0",
+		"highlight.js": "^11.8.0",
+		"prettier": "^2.8.8",
+		"prettier-plugin-svelte": "^2.10.1",
+		"sass": "^1.62.1",
+		"svelte": "^4.0.0",
+		"svelte-check": "^3.4.3",
+		"svelte-preprocess": "^5.0.3",
+		"svelte2tsx": "^0.6.14",
+		"tslib": "^2.5.2",
+		"typescript": "^5.0.4",
+		"vite": "^4.3.8"
+	},
+	"peerDependencies": {
+		"ag-grid-community": "^28 || ^29 || ^30",
+		"svelte": "^3 || ^4"
+	},
+	"scripts": {
+		"dev": "vite dev",
+		"build": "vite build",
+		"package": "svelte-package",
+		"preview": "vite preview",
+		"test": "playwright test",
+		"check": "svelte-check --tsconfig ./tsconfig.json",
+		"check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
+		"lint": "prettier --check --plugin-search-dir=. . && eslint .",
+		"format": "prettier --write --plugin-search-dir=. .",
+		"release": "commit-and-tag-version"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/brandonp-ais/ag-grid-svelte.git"
```

Comment:
@brandonp-ais just curious, shouldn't these stay pointing to MichaelKim's repo for the PR?

Reply:
I don't know tbh. This PR was poorly done on my part and I'll close it. But the process for updating to svelte-4 is outlined in the issue.